scripts/troubleshooting.sh

```diff
@@ -39,21 +62,49 @@ detect_distro() {
 
 panel_logs(){
   if [ -f "/var/www/pterodactyl/storage/logs/laravel-$(date +%F).log" ]; then 
-    panel_log="$(tail -n 100 /var/www/pterodactyl/storage/logs/laravel-"$(date +%F)".log | nc bin.ptdl.co 99)"
+    panel_log="$(nc termbin.com 9999 < /var/www/pterodactyl/storage/logs/laravel-"$(date +%F)".log | tr -d '\0')"
```

Comment:
Would it be feasible to use something else than termbin? Would it also be possible to not hardcode it?

Reply:
Sure, but do you mean like ENV var or just have a variable in the script?

Reply:
An environment variable that would be specified in `.env` (which is read by `docker-compose`). I presume, without reading the code too closely, that the bot webserver actually reads the `.sh` file so you could probably fill in variables at that step?

Reply:
When I load the script I can just replace the URL with the one in the ENV var. It is pretty straight forward.

Reply:
Fixed